.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG] "
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
What these explains?

Reply:
How to reproduce the error, i.e., what other person should do to get that error. Series of steps that will result in error.

Reply:
I am not getting.
Explain in detail.
with example